packages/mdc-textfield/foundation.js

```diff
@@ -65,6 +75,10 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
     this.adapter_.registerInputBlurHandler(this.inputBlurHandler_);
     this.adapter_.registerInputInputHandler(this.inputInputHandler_);
     this.adapter_.registerInputKeydownHandler(this.inputKeydownHandler_);
+    this.adapter_.registerTransitionEndHandler(this.transitionEndHandler_);
+    ['mousedown', 'touchstart'].forEach((evtType) => {
```

Comment:
Should we use focus trap for this? MDC Ripple Foundation also looks at 'pointerdown', 'keydown', and 'focus'. Should we register on those event types as well?